landr.conf.js

```diff
@@ -0,0 +1,62 @@
+const prepAssets = require('./www/prepAssets')
+
+module.exports = {
+  theme: 'landr-theme-basic',
+  middleware: (store, action, next) => {
+    if (action.type === 'ADD_RELEASE') {
+      // intercept all releases and add pretty labels to assets
+      action.payload = prepAssets(action.payload)
+    }
+    return next(action)
+  },
+  settings: {
+    analytics: {
+      mixpanelToken: '9d6bc43e4d64eb3bd64922c969e2955f',
+      gosquaredId: 'GSN-954701-N',
+      gaSite: 'etcher.io',
+      gaId: 'UA-45671959-2'
+    },
+    theme: {
+      colors: {
+        primary: '#1496E1'
+      }
+    },
+    lead: 'Burn. Better.',
+    features: [
+      {
+        'title': 'Validated Burning',
+        'image': 'sd.png',
+        'description': 'No more writing images on corrupted cards and wondering why your device isn\'t booting.'
+      },
+      {
+        'title': 'Hard Drive Friendly',
+        'image': 'hd.png',
+        'description': 'Makes drive selection obvious to avoid wiping your entire hard-drive'
+      },
+      {
+        'title': 'Beautiful Interface',
+        'image': 'simple.png',
+        'description': 'Who said burning SD cards has to be an eyesore.'
+      },
+      {
+        'title': 'Open Source',
+        'image': 'open-source.png',
+        'description': 'Made with JS, HTML, node.js and <a target="_blank" href="http://electron.atom.io/">Electron</a>. Dive in and contribute!'
+      },
+      {
+        'title': 'Cross Platform',
+        'image': 'cross-platform.png',
+        'description': 'Works for everyone,<br/>no more complicated install instructions.'
+      },
+      {
+        'title': 'More on the way',
+        'image': 'coming-soon.png',
+        'description': '50% faster burns, simultaneous writing for multiple drives.<br/><a target="_blank" href="https://github.com/resin-io/etcher/milestones">View our roadmap</a>'
+      }
+    ],
+    motivation: [
```

Comment:
This should be automatically extracted from the repo. Can you create a new PR that adds this as a "Why" section in the README, and then we can pretend it comes from there?